amqpstorm/tests/unit/io/io_exception_tests.py

```diff
@@ -36,6 +36,15 @@ def test_io_receive_raises_socket_error(self):
             connection.check_for_errors
         )
 
+    def test_io_receive_raises_ssl_want_read_error(self):
```

Comment:
Add this line on the top of this class.
```
    def configure(self):
        self.disable_logging_validation()
```
It will disable the check for unexpected log messages. This code path previously had no warning loggs during the test.

Reply:
Thanks for the hint.

After looking at the functional tests, I added `if self._running.is_set():` in front of the log message, so we only see it if this code path actually stops the io thread, and not if it appears concurrently with the `IO.close()` function. 
As the unit tests don't set `running`, the log messages there went away as well...

Reply:
I like that.